base/file.jl

```diff
@@ -515,7 +515,7 @@ function sendfile(src::AbstractString, dst::AbstractString)
         src_file = open(src, JL_O_RDONLY)
         src_open = true
         dst_file = open(dst, JL_O_CREAT | JL_O_TRUNC | JL_O_WRONLY,
-             S_IRUSR | S_IWUSR | S_IRGRP | S_IWGRP| S_IROTH | S_IWOTH)
+             S_IRUSR | S_IWUSR | S_IRGRP | S_IWGRP | S_IROTH | S_IWOTH)
```

Comment:
Looks like in other cases we've been making this a `mode::Unsigned = 0o666` kw-arg, which also seems good